corehq/apps/sms/templates/sms/email/new_sms_user.html

```diff
@@ -4,10 +4,6 @@
     A new user {{ username }} registered for project {{ domain }} through SMS.
   </p>
 
-  <p>
-    Here is the full content of their registration message: {{ message_text }}
-  </p>
-
```

Comment:
:+1:

Reply:
@mkangia do you think this is a sufficient change? I can add an "SMS view" later if it is needed.

Reply:
Yup. I feel good to wait on adding the SMS view, until its requested

Reply:
Great! If so, would you be OK approving this PR then?

Reply:
oh sorry, I was waiting for your response on other comments though, which were non-blocking.